xapian-applications/omega/omegatest

```diff
@@ -432,6 +432,15 @@ printf '$last$if{$cgi{ERR},$seterror{$cgi{ERR}}}$msize$if{$error,!$error!}' > "$
 testcase '11' P=text
 testcase '11!boo!' P=text ERR=boo
 
+# Feature tests for $hash{}
+printf '$hash{$cgi{text},md5}' > "$TEST_TEMPLATE"
+testcase '098f6bcd4621d373cade4e832627b4f6' text="test"
+testcase 'b4c216e4da73d1d01277ef46d0514821' text="simple query"
+testcase 'd41d8cd98f00b204e9800998ecf8427e' text=""
+
+printf '$hash{$cgi{text},sha-1}' > "$TEST_TEMPLATE"
+testcase '' text="test"
```

Comment:
For testing the "not a valid hash name" case, `sha-1` is a bit of a confusing choice as it looks like the name of a valid hash.  Better to pick something that's clearly not a valid hash to the reader, e.g. like `bogus` or `notavalidhash`.